### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 
 ## Project status
 [![Build Status](https://travis-ci.org/genonbeta/TrebleShot.svg)](https://travis-ci.org/genonbeta/TrebleShot)
 [![Translation Status](https://hosted.weblate.org/widgets/trebleshot/-/svg-badge.svg)](https://hosted.weblate.org/engage/TrebleShot/)
 
 ## Donate
-If you value this work, you can help it. There are only a few methods that work in Turkey,
-you can use Bitcoin or install Google Play version to do so (In App Billing method) (taxes will apply).
+If you value this work, you can help it along. Only a few methods of monetary contribution work in Turkey,
+you can use Bitcoin or install the Google Play version to do so (In App Billing method) (taxes will apply).
```
Comment (comradekingu) [inline README.md]: What does " to do so (In App Billing method)" mean?

Comment (velitasali) [reply README.md]: It states that they can donate via Google Pay on their Android devices.

Comment (comradekingu) [inline README.md]:
```suggestion
you can use Bitcoin or install the Google Play version (using App Billing) to do so (taxes will apply).
```

Comment (velitasali) [reply README.md]: I can't seem to be able to commit.

**Pull request merged**